Automattic/simplenote-ios
# Fixing Swipe to Dismiss

### Fix
In this PR we're fixing a broken logic that prevented the Editor from being dismissed via Swipe gestures, when Markdown enabled notes were onScreen.

@frosty Apologies to bug you, yet again. Sending you this tiny PR over, for historic reasons.
The header in InteractivePushPopAnimationController reads `08/10/2015`. Time goes by quickly, doesn't it?

Thanks in advance!!

Closes #491


### Test
1. Log into your account
2. Add a new note
3. Enable Markdown in it

- [ ] Verify that Swipe from Right to Left displays the Markdown Preview UI
- [ ] Verify that Swipe from Left to Right dismisses the Markdown Preview UI
- [ ] Verify that you can swipe from the left edge of the screen, **anywhere**, and the Editor gets dismissed

### Release
`RELEASE-NOTES.txt` was updated in 62bbfed with:

> Fixed a bug that caused Swipe to Dismiss gesture not to properly work in the Editor
